lms/envs/aws.py

```diff
@@ -590,6 +590,8 @@
             'schedule': datetime.timedelta(hours=ENV_TOKENS.get('THIRD_PARTY_AUTH_SAML_FETCH_PERIOD_HOURS', 24)),
         }
 
+    SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS = ENV_TOKENS.get('SOCIAL_AUTH_RESPECT_X_FORWARDED_HEADERS')
```

Comment:
@mattdrayer `False` would make sense, but None is still falsy + indicates that nothing is actually specified in ENV_TOKENS, so we might want to change the default in future.